classes/Hook.php

```diff
@@ -480,7 +486,7 @@ public static function getHookModuleList()
         }
 
         $results = Db::getInstance(_PS_USE_SQL_SLAVE_)->executeS(
-            'SELECT h.id_hook, h.name as h_name, title, description, h.position, hm.position as hm_position, m.id_module, m.name, active
+            'SELECT h.id_hook, h.name as h_name, title, description, h.position, hm.position as hm_position, m.id_module, m.name, m.active
```

Comment:
Oh ? there was already a `active` field inside this SQL query 😮  ?

Reply:
Haha I got it after 😄  nice one ! it's a good one 👍

Reply:
Yes this is the active field of the modules, this can cause ambiguity problems with active field hook.

Reply:
First I was "eh ? the field already exists ?"
Then "Ah, no, he replaced the module active field by his active field, it's bad"
Then "Ah, no, he prefixed it which means ... ah, it means everything's good" 😄 

Nice one !